src/java/neroxis/generator/ImageGenerator.java

```diff
@@ -0,0 +1,283 @@
+            greenStrength = Float.parseFloat(arguments.get("green"));
+        }
+
+        if (arguments.containsKey("blue")) {
+            blueStrength = Float.parseFloat(arguments.get("blue"));
+        }
+
+        if (arguments.containsKey("level-of-detail")) {
+            levelOfDetail = Integer.parseInt(arguments.get("level-of-detail"));
+        }
+
+        if (arguments.containsKey("max-feature-size")) {
+            maxFeatureSize = Integer.parseInt(arguments.get("max-feature-size"));
+        }
+    }
+
+    public void generateCustomBrushes(int size, int numberToGenerate) throws IOException {
+
+        for (int i = 0; i < numberToGenerate; i++) {
+            int brushListLength = Brushes.GENERATOR_BRUSHES.size();
+            int reducedSize = size * 2 / 3;
+            int variationDistance = StrictMath.max(size - reducedSize - 3, 0);
+            int center = size / 2;
+            int mountainsBrushSize = size / 10;
+            Random random = new Random();
+
+            String brush1 = Brushes.GENERATOR_BRUSHES.get(random.nextInt(brushListLength));
+            String brush2 = Brushes.GENERATOR_BRUSHES.get(random.nextInt(brushListLength));
+            String brush3 = Brushes.GENERATOR_BRUSHES.get(random.nextInt(brushListLength));
+            String brush4 = Brushes.GENERATOR_BRUSHES.get(random.nextInt(brushListLength));
+            String brush5 = Brushes.GENERATOR_BRUSHES.get(random.nextInt(brushListLength));
+
+            BinaryMask base = new BinaryMask(size, random.nextLong(), new SymmetrySettings(Symmetry.NONE, Symmetry.NONE, Symmetry.NONE));
+
+            base.combineBrush(new Vector2f(center + random.nextInt(variationDistance) - random.nextInt(variationDistance),
+                    center + random.nextInt(variationDistance) - random.nextInt(variationDistance)), brush1, random.nextFloat(), 1f, reducedSize);
+            base.combineBrush(new Vector2f(center + random.nextInt(variationDistance) - random.nextInt(variationDistance),
+                    center + random.nextInt(variationDistance) - random.nextInt(variationDistance)), brush2, random.nextFloat(), 1f, reducedSize);
+            base.combineBrush(new Vector2f(center + random.nextInt(variationDistance) - random.nextInt(variationDistance),
+                    center + random.nextInt(variationDistance) - random.nextInt(variationDistance)), brush3, random.nextFloat(), 1f, reducedSize);
+            base.combineBrush(new Vector2f(center + random.nextInt(variationDistance) - random.nextInt(variationDistance),
+                    center + random.nextInt(variationDistance) - random.nextInt(variationDistance)), brush4, random.nextFloat(), 1f, reducedSize);
+            base.combineBrush(new Vector2f(center + random.nextInt(variationDistance) - random.nextInt(variationDistance),
+                    center + random.nextInt(variationDistance) - random.nextInt(variationDistance)), brush5, random.nextFloat(), 1f, reducedSize);
```

Comment:
Uh what is the point of adding and subtracting a random?

Reply:
it yields a desirable spread of variation within the range of center +/- variationDistance

Reply:
You could just use random.nextInt(variationDistance * 2) - variationDistance

Reply:
also random.nextGaussian

Reply:
but depends on if you want the uniform distribution or not

Reply:
I don't want uniform distrubution